### PR TITLE
enh(Resources/Details/TimeLine):Timeperiod-group-button-and-custom-period-selectors-have-different-heights

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
put one height for timeLine section
**some screens :** 

> **spanish**
timeLine2

> **anglais** 
timeLine1

### CODE REVIEW

**Comment by bdauria** [comment]
@Noha-ElAbrouki can we get a screenshot for @Wedierre ?

**Comment by bdauria** [comment]
@Noha-ElAbrouki is there a way to display the custom perdio button in one line for all the cases?

**Comment by Wedierre** [comment]
@bdauria @Noha-ElAbrouki 
If I refer to the screen shot above it seems fine to me. 
Be careful to edit:
- we must have the same space at the top and bottom between the outline of the block and the text (see capture).
- align "From and To"

Other points noted in the panel:
- Difficulty reading the black text "UNKNOWN" on a grey background "do a white essay"  ?
Center it in height in the chips.
- Center the blue icons on the middle of the tiles under the heading "Today"
- In the tile align with the center the date and the "UNKNOWN" status chips
capture panel

**Comment by bdauria** [comment]
@Noha-ElAbrouki Can you show us a screenshot when the panel is expanded?

File: www/front_src/src/Resources/Graph/Performance/TimePeriods/CustomTimePeriodPickers.tsx [hunk 1]
```diff
@@ -55,9 +55,17 @@ const useStyles = makeStyles((theme) => ({
...
   },
   compactFromTo: {
-    columnGap: theme.spacing(0.5),
-    display: 'grid',
-    grid: 'repeat(2, min-content) / min-content auto',
+    display: 'flex',
+    flexDirection: 'column',
+    padding: theme.spacing(0.5, 0, 0.5, 0),
+  },
+  containerTime: {
```
Comment (bdauria) [inline www/front_src/src/Resources/Graph/Performance/TimePeriods/CustomTimePeriodPickers.tsx:62]:
```suggestion
  timeContainer: {
```

File: www/front_src/src/Resources/Graph/Performance/TimePeriods/CustomTimePeriodPickers.tsx [hunk 2]
```diff
@@ -68,6 +76,11 @@ const useStyles = makeStyles((theme) => ({
     display: 'grid',
     gridTemplateColumns: 'repeat(4, auto)',
   },
+  labelDate: {
```
Comment (bdauria) [inline www/front_src/src/Resources/Graph/Performance/TimePeriods/CustomTimePeriodPickers.tsx:79]:
```suggestion
  dateLabel: {
```

**Comment by Noha-ElAbrouki** [comment]
> @Noha-ElAbrouki Can you show us a screenshot when the panel is expanded?

https://user-images.githubusercontent.com/97687698/173780271-0103fb43-3c3d-4572-97f7-5306dc2d9be2.mp4

**Comment by Noha-ElAbrouki** [comment]
> @Noha-ElAbrouki Can you show us a screenshot when the panel is expanded?

https://user-images.githubusercontent.com/97687698/173782736-508a75d7-2d57-4470-b394-e7a200740371.mp4

**Comment by bdauria** [comment]
@Wedierre What do you think?

**Comment by Wedierre** [comment]
I think it’s okay.
responsive works well.
there’s a bit of latency in the menu under "Timeline", is that normal?

**Review by bdauria — APPROVED** [review]

**Comment by bdauria** [comment]
> I think it’s okay. responsive works well. there’s a bit of latency in the menu under "Timeline", is that normal?

That's the current behaviour, but we could work that out in a separate context

**Comment by Thebarda** [comment]
> I think it’s okay. responsive works well. there’s a bit of latency in the menu under "Timeline", is that normal?

I don't know. where it come from because it's fully managed by Material UI. But I bet, it's related to animations

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**